lib/VisualBuilder.js

```diff
@@ -182,6 +190,33 @@ class VisualBuilder extends EventEmitter {
     }
 
     /**
+     * Updates the capabilities content at build time
+     * For example for R Custom Visuals, we take the content of a script.yyy file 
+     * and place the content of this file inside the capabilities
+     * @private
+     */
+    _updateCapabilities(capabilities) {
+        if (capabilities && 
+            capabilities.dataViewMappings && 
+            capabilities.dataViewMappings.length === 1 && 
+            capabilities.dataViewMappings[0].scriptResult) {
+            let scriptResult = capabilities.dataViewMappings[0].scriptResult;
+            if (scriptResult.script.scriptProviderDefault && !scriptResult.script.scriptSourceDefault) {
+                ConsoleWriter.info("Updating visual capabilities scriptResult.script.scriptProviderDefault");
```

Comment:
I don't think we need to output about this. The whole visual build should either succeed or fail (Which would include the R portion). If you really want this granular output you could maybe emit an event or something, but I don't think it's needed.